### PR TITLE
adds hacktoberfest 2019 project (+)

### PR DESCRIPTION
**Pull request by connerk**
additional:
bumps  browser-sync (security vulnerabilities)
update featured projects

### CODE REVIEW

File: _projects/Hacktoberfest2019/index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+sort-order: 10
+
+# modify these to be specific to your project
+# mandatory
+title: Hacktoberfest 2019
+
+# optional - if not used, comment out using '#' (i.e. date:)
+hackers: [Kevin Conner]
+# date:
+abstract: if you provide an abstract, this is the short description that will show up in the project index
```
Comment (pcunning) [inline _projects/Hacktoberfest2019/index.md]: needs to be updated - is on homepage.

File: _projects/Hacktoberfest2019/index.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+keywords: [Hacktoberfest, Meetup, Event]
+thumbnail: ./Hacktoberfest_19_Events_500x500.png
+link: https://hacktoberfest.digitalocean.com/
+
+---
+
+A celebration of open source software through the month of October.
+<!-- more -->
+
+<img src="LAD-ad.jpg">
```
Comment (pcunning) [inline _projects/Hacktoberfest2019/index.md]: can we put text in here so it's more accessible?

Comment (connerk) [reply _projects/Hacktoberfest2019/index.md]: yes, I intend to convert it to HTML soon but wanted to get something quick up in ordered to get registered.

File: _projects/Hacktoberfest2019/index.md (hunk 1 above)
Comment (connerk) [reply _projects/Hacktoberfest2019/index.md]: fixed

**Pull request merged**